src/components/Checkout/MonthlyGoodCTA.vue

```diff
@@ -61,26 +88,50 @@
 
 <script>
 import numeral from 'numeral';
+import { validationMixin } from 'vuelidate';
+import { required, minValue, maxValue } from 'vuelidate/lib/validators';
 
 import KvButton from '@/components/Kv/KvButton';
 import KvIcon from '@/components/Kv/KvIcon';
 import KvDropdownRounded from '@/components/Kv/KvDropdownRounded';
+import KvCurrencyInput from '@/components/Kv/KvCurrencyInput';
 
 import loanGroupCategoriesMixin from '@/plugins/loan-group-categories';
 
+/**
```

Comment:
I'll admit this is kind of strange

Reply:
yea I think the landing form only has the total/overall 10k limit... 8500 could be ok I suppose for a near term guard.